docs/src/pages/components/drawers/SwipeableEdgeDrawer.tsx

```diff
@@ -18,6 +19,11 @@ interface Props {
   window?: () => Window;
 }
 
+const Root = styled('div')(({ theme }) => ({
```

Comment:
https://next--material-ui.netlify.app/components/drawers/#swipeable-edge

![image](https://user-images.githubusercontent.com/42154031/115099984-f5accb00-9f0f-11eb-9885-ebe1a59142bf.png)

Reply:
The new version doesn't look right. The lower the background is, the darker is should be: https://material.io/design/color/dark-theme.html#ui-application.